pages/getting-started/hosting-domains/hosting-on-hover.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Hosting on a custom domain - Hover
+description: "A step-by-step guide: setting up a custom domain to host your newsletter with Hover"
+---
+
+import Layout from "../../../components/Layout";
+import Notice from "../../../components/Notice";
+
+export const meta = {
+  title: "Hosting on a custom domain - Hover",
+  description:
+    "A step-by-step guide: setting up a custom domain to host your newsletter with Host",
```

Comment:
"with Host" — "with Hover"?

Reply:
Oh hey, I think this is the thing that turned into a big snaggle-weed.  It is fixed now.